tests/test_crs.py

```diff
@@ -201,7 +210,7 @@ def test_is_same_crs():
     assert crs1 != crs2
 
     wgs84_crs = CRS.from_string('+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs')
-    assert crs1 == wgs84_crs
+    assert crs1 != wgs84_crs
```

Comment:
🤔 does this means the behaviour will change, or this is more a `bug fix`?

Reply:
that is a good question and is a question for the maintainers, I think the equality test before this PR was not working as intended. If the desire is for the axis order to not be ignored by default, then the behavior in the test above was a bug, because depending on the exact situation these proj4 strings could resolve as 4326 via `to_epsg()` and the equality check would first attempt to convert to epsg codes. Seemingly the matches criteria is a bit looser by default when converting to epsg codes. For this test to keep that behavior one would use the new `.equals` method with the ignore flag enabled instead. If however we want to keep the "behavior" as is for this case and the other test, the ignore flag should be enabled by default, and to my eye epsg:4326 and crs84 _are_ equivalent except for the axis order.

Reply:
@sgillies @snowman2, what do you think?